recipes/vsnp3/meta.yaml

```diff
@@ -37,7 +37,7 @@ requirements:
     - regex >=2.5.110
     - samtools >=1.14
     - seqkit >=2.1.0
-    - vcflib =1.0.9
+    - vcflib =1.0.1
```

Comment:
Do you know why this needs to be downgraded?

Reply:
I went through all the versions. It wasn’t until 1.0.1 that it worked on both RedHat and MacOS. 1.0.9 worked on the MacOS, but not on RedHat. I had to go back to 1.0.1 to get it to work on both.

I’m noticing now that I was getting one of two errors on testing. See below.

`vcfuniq -h`                                                                                                                                                                                                                                         
vcfuniq: error while loading shared libraries: libtabixpp.so.0: cannot open shared object file: No such file or directory

`vcfuniq -h`                                                                                                                                                                                                                                               
vcfuniq: symbol lookup error: /home/tstuber/.conda/envs/vsnp3-test1/bin/../lib/libvcflib.so.1: undefined symbol: wavefront_align

Reply:
Also tried leaving off a version specification, but that was failing too.

Reply:
@BiocondaBot please add label